app-editors/ghex/ghex-43.0.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="GNOME hexadecimal editor"
+HOMEPAGE="https://wiki.gnome.org/Apps/Ghex"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.gnome.org/GNOME/ghex.git"
+	SRC_URI=""
+else
+	KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~riscv ~x86 ~amd64-linux ~x86-linux"
+fi
+
+LICENSE="GPL-2+ FDL-1.1+"
+IUSE="gtk-doc test"
+RESTRICT="!test? ( test )"
+SLOT="4"
+
+RDEPEND="
+	>=dev-libs/glib-2.74.0:2
+	>=gui-libs/gtk-4.4.0:4
+	>=gui-libs/libadwaita-1.2.0:1
```

Comment:
We want to keep the versions in the ebuild in sync with those required in `meson.build`. It makes things easier in version bumps to match up changes.

````diff
+glib_ver = '>= 2.68.0'
 gmodule_dep = dependency('gmodule-2.0')
-gio_dep = dependency('gio-2.0', version: '>= 2.66.0')
-gtk_dep = dependency('gtk4', version: '>= 4.0.0')
+glib_dep = dependency('glib-2.0', version: glib_ver)
+gio_dep = dependency('gio-2.0', version: glib_ver)
+gtk_dep = dependency('gtk4', version: '>= 4.4.0')
+adw_dep = dependency('libadwaita-1')
````